src/index.html

```diff
@@ -5,10 +5,52 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin = "anonymous" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home"  >
+        <img src="images/logo.png"
+             alt ="moyo-logo"
+             class = "logo"
```

Comment:
You need to check and fix it everywhere
<img width="1038" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/a2f75fb0-41a4-44fd-8072-fff86e408318">
```suggestion
        <img
          src="images/logo.png"
          alt="moyo-logo"
          class="logo"
```